graphql/certificate_of_conduct/fields.ts

```diff
@@ -0,0 +1,21 @@
+// eslint-disable-next-line camelcase
+import {Certificate_of_conduct, Course, Screener, Student} from "../generated";
+import { Authorized, FieldResolver, Resolver, Root } from "type-graphql";
+import { prisma } from "../../common/prisma";
+import { Role } from "../authorizations";
+
+
+// eslint-disable-next-line camelcase
+@Resolver(of => Certificate_of_conduct)
+export class ExtendedFieldsCertificateOfConductResolver {
+    @FieldResolver(returns => [Student])
+    @Authorized(Role.ADMIN)
+    // eslint-disable-next-line camelcase
+    async student(@Root() certificate: Certificate_of_conduct) {
+        return await prisma.student.findMany({
```

Comment:
Shouldn't this be `findUnique`?

Reply:
Logically, it doesnt make a different since both are unique anyways but syntactically yes. I'll update this.